earthpy/clip.py

```diff
@@ -143,17 +143,23 @@ def clip_shp(shp, clip_obj):
             " valid geometry column"
         )
 
+    #if clip_obj["geometry"].iloc[0].type != "Polygon":
+        #raise AttributeError("Trying to clip an object with something other then a polygon.")
+
     if not any(shp.intersects(clip_obj.unary_union)):
         raise ValueError("Shape and crop extent do not overlap.")
-
-    # Multipolys / point / line don't clip properly
-    if "Multi" in str(clip_obj.geom_type) or "Multi" in str(shp.geom_type):
-        raise ValueError(
-            "Clip doesn't currently support multipart geometries. Consider "
-            "using .explode to create unique features in your GeoDataFrame"
-        )
-
-    if shp["geometry"].iloc[0].type == "Point":
+    # Multipolys / point / line clip differently then non-multi features.
+    if "Multi" in str(shp.geom_type):
+        if shp["geometry"].iloc[0].type == "Point" or shp["geometry"].iloc[0].type == "MultiPoint":
```

Comment:
@nkorinek  this looks like it is testing for point geometry here. you'd want to clip multi points in the point clip function rather than here.